src/components/Payment.js

```diff
@@ -140,6 +141,8 @@ const Payment = ({ notify, pushChanges, values, controls }) => {
           }}
         </ThirdPartyIntegration>
       )}
+      </Section.Content>
+      <Section.Content separator>
       {RuntimeSdk.brandSettings.setup.payment.includes("Razorpay") && (
         <Razorpay notify={notify} />
```

Comment:
Can we use `ThirdPartyIntegration` component for `Razorpay` as well? like we have done for other payment methods.